pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -525,7 +555,9 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
 
         // Since this can only be called by root, which is always a canceler for all scheduled executions, we don't
         // bother creating any new cancelers.
-        return _scheduleWithDelay(actionId, address(this), data, getRootTransferDelay(), executors);
+        uint256 scheduledExecutionId = _scheduleWithDelay(address(this), data, getRootTransferDelay(), executors);
+        emit RootChangeScheduled(actionId, scheduledExecutionId);
```

Comment:
I know that `actionId` is always the same but I say we keep it for consistency but it is a pretty weak opinion of mine.